iocore/net/P_SNIActionPerformer.h

```diff
@@ -261,14 +264,14 @@ class TLSValidProtocols : public ActionItem
   TLSValidProtocols() : protocol_mask(max_mask) {}
   TLSValidProtocols(unsigned long protocols) : unset(false), protocol_mask(protocols) {}
   int
-  SNIAction(Continuation *cont, const Context & /* ctx */) const override
+  SNIAction(TLSSNISupport *snis, const Context & /* ctx */) const override
   {
     if (!unset) {
-      auto ssl_vc = dynamic_cast<SSLNetVConnection *>(cont);
+      auto ssl_vc = dynamic_cast<SSLNetVConnection *>(snis);
```

Comment:
Looking through this, I realize there is no value to doing the dynamic_cast here as opposed to a static_cast since we don't bother to check the return value.   That is an issue that predates this PR, so I'll put up a separate PR for that.